config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -67,6 +67,38 @@ presubmits:
               cpu: 4
               memory: "36Gi"
 periodics:
+  - interval: 1h
+    name: ci-kubernetes-unit
+    annotations:
+      fork-per-release: "true"
+      testgrid-num-failures-to-alert: '3'
+      testgrid-alert-stale-results-hours: '24'
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: sig-release-master-informing
```

Comment:
this really ought to be in blocking, but given that the job doesn't exist yet and we had a gap in this versus the bazel based previous job ...

Reply:
agree